registry/testcases/unit_testcases/test_organization_publisher_service.py

```diff
@@ -0,0 +1,432 @@
+import unittest
+from datetime import datetime
+from unittest.mock import Mock, patch
+from uuid import uuid4
+
+from common.utils import validate_dict
+from registry.application.services.organization_publisher_service import OrganizationPublisherService, org_repo
+from registry.constants import OrganizationStatus, Role
+from registry.domain.models.organization import Organization as DomainOrganization
+from registry.domain.factory.organization_factory import OrganizationFactory
+from registry.infrastructure.models import Organization, OrganizationMember, OrganizationState, Group, \
+    OrganizationAddress
+
+ORIGIN = "PUBLISHER_DAPP"
+ORG_PAYLOAD_REQUIRED_KEYS = ["org_id", "org_uuid", "org_name", "origin", "org_type", "metadata_ipfs_hash",
+                             "description", "short_description", "url", "contacts", "assets",
+                             "mail_address_same_hq_address", "addresses", "duns_no"]
+
+
+class TestOrganizationPublisherService(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    @patch("common.ipfs_util.IPFSUtil", return_value=Mock(write_file_in_ipfs=Mock(return_value="Q3E12")))
+    @patch("common.boto_utils.BotoUtils", return_value=Mock(s3_upload_file=Mock()))
+    def test_add_organization_draft(self, mock_boto, mock_ipfs):
+        username = "dummy@dummy.com"
```

Comment:
Can we make this test case a little shorter?
We should dump test data in some file or test DB, making test cases short and easier to understand